opencensus/stats/measure_to_view_map.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.tags import tag_map
+from opencensus.stats.measurement import Measurement
+from opencensus.stats import measurement
+from opencensus.stats.view import View
+from opencensus.stats.view_data import ViewData
+from datetime import datetime
+import logging
+
+
+class MeasureToViewMap(object):
+
+    def __init__(self):
+        self._map = {}
```

Comment:
This should be a multi map ([Java counterpart](https://github.com/census-instrumentation/opencensus-java/blob/master/impl_core/src/main/java/io/opencensus/implcore/stats/MeasureToViewMap.java#L58)).